src/frontend/components/ConceptContainer.vue

```diff
@@ -16,7 +16,7 @@
               data-toggle="tooltip"
             ><b-icon-info-circle-fill /></a></sup>
             <sup><a
-              href="https://dataarc-demo.readthedocs.io/en/latest/#getting-started"
+              href="https://dataarc-demo.readthedocs.io/en/latest/dataarc-tool-help.html#conceptual-filter-how-do-i-use-create-a-conceptual-filter"
```

Comment:
Would it be possible to centralize these in a single js file and prototyped to the Vue instance? That way if these change (given our luck) it's an easy update.